v20.1/enable-node-map.md

```diff
@@ -123,7 +125,8 @@ Insert the approximate latitudes and longitudes of each region into the `system.
   ('region', 'eu-west-1', 53.142367, -7.692054);
 ~~~
 
-{{site.data.alerts.callout_info}}The <b>Node Map</b> will not be displayed until all regions are assigned the corresponding latitudes and longitudes. {{site.data.alerts.end}}
+{{site.data.alerts.callout_info}}
+The Node Map will not be displayed until each region is assigned a corresponding latitude and longitude. {{site.data.alerts.end}}
```

Comment:
The closing tag of this callout could be on the next line (not important, but hey)